gemrb/GUIScripts/InventoryCommon.py

```diff
@@ -950,4 +951,18 @@ def AbilitiesItemWindow ():
 	Window.ShowModal (MODAL_SHADOW_GRAY)
 	return
 
+def HookKeyPresses():
+	Window = GUIINV.InventoryWindow
+	Window.SetKeyPressEvent(SwitchPC)
+	return
```

Comment:
This function looks redundant. The binding will always happen in a context where we know the window we want to work with, right? So `InventoryWindow.SetKeyPressEvent (SwitchPC)` is better than going through another layer.